tasks/filebeat_override.yml

```diff
@@ -0,0 +1,43 @@
+---
+
+- name: Parse ProxySQL filebeat input
+  set_fact:
+    proxysql_input:
+      filebeat:
+        inputs:
```

Comment:
I do not really get why this structure is like this. The list `inputs` has the first item, without key, but it is a list. and the rest of the elements. Maybe it makes sense, but I would not expect the filebeat config to fit in here :thinking: . Do you have by any chance the expected outcome?

Reply:
nvm, the format is like that in filebeat :shrug:

Reply:
Hi @bvcelari, thank you for reviewing! :)
In order to do the override I need to combine the two dictionaries. This requires the new input to be nested in the same fashion as the original dict.